src/streamlit_pydantic/ui_renderer.py

```diff
@@ -143,11 +151,22 @@ def render_ui(self) -> Dict:
                 # Set property key as fallback title
                 property["title"] = _name_to_title(property_key)
 
+            # if there are instance values, add them to the property dict
+            if instance_dict is not None:
+                instance_value = instance_dict.get(property_key)
+                if instance_value in [None, ""]:
+                    instance_value = instance_dict_by_alias.get(property_key)
+                if instance_value not in [None, ""]:
+                    property["init_value"] = instance_value
+                    property["instance_class"] = str(
+                        type(getattr(self._input_class, property_key))
```

Comment:
This currently leads to a problem if `alias` is used. Would the type of `instance_value` be sufficient here? Otherwise, we can also just check if attr exists, and otherwise set to empty or `None` (-> so the type-based matching wouldn't work if alias is used). 

<img width="714" alt="image" src="https://user-images.githubusercontent.com/2852129/179511952-de0c4a21-b802-45be-83f8-92a481f2613e.png">

Reply:
argh! Sorry I can't believe I let that slip through! 

I've now put a guard in place there and a `TODO` about aliases.

Unfortunately `instance_value` is only a dict so getting the original type is a bit more involved.

A possible approach to making aliases work here (and perhaps something that would make this section a bit more robust in general) would be to iterate through all of the the class `__fields__` list directly to build up the property metadata rather than accessing via key.  That's a bigger change than I'd be comfortable making as part of this PR though.

Reply:
Perfect, thank you!